elyra/pipeline/processor_kfp.py

```diff
@@ -37,6 +38,30 @@ class KfpPipelineProcessor(PipelineProcessor):
     def type(self):
         return self._type
 
+    def get_auth_session_cookie(self, url, api_endpoint_username, api_endpoint_password):
```

Comment:
and simplify the variable names - no need to know their context within the function.
```suggestion
    def get_auth_session_cookie(self, url, username, password):
```